cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -99,6 +99,41 @@ const char *log_level(long level)
 #define LOG_SIZE    512
 static char log_buff[LOG_SIZE];
 
+
+/** Log as a backup, if python isn't working */
+static void gpi_log_native_v(const char *name, long level, const char *pathname, const char *funcname, long lineno, const char *msg, va_list argp) {
```

Comment:
Can you please wrap that line and place the `{` on a separate line for consistent style within this file?